.github/workflows/web-actions.yml

```diff
@@ -53,21 +53,25 @@ jobs:
       - name: Run tests and generate coverage file
         run: npm run ci:test
 
-      - name: Setup LCOV
-        uses: hrishikesh-kadam/setup-lcov@v1
-      - name: Report code coverage
-        uses: zgosalvez/github-actions-report-lcov@v3
-        with:
-          coverage-files: ./app/lcov*.info
-          minimum-coverage: 40
-          artifact-name: code-coverage-report
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          working-directory: ./app
-          update-comment: true
+      # - name: Setup LCOV
```

Comment:
Don't comment out code. We use a version control system. We can get it back when we need to. You can just delete this.